Quality control for human biomolecular data page (#290)
elixir-europe/infectious-diseases-toolkit: quality-control/human-biomolecular-data.md

---
title: Human biomolecular data
description: This page explains in detail the importance of quality control for human biomolecular data in the context of infectious diseases, as well as explains the steps to follow.

> contributors: [Aina Jené Cortada, Mireia Marin Ginestar]

Author: oh sorry that I missed them. Can you add Clementina to the Contributors yaml? I don't know her details.

> contributors: [Aina Jené Cortada, Mireia Marin Ginestar]

Author: Added! :)

- **Collaboration and Data Sharing**: During an outbreak, collaboration and data sharing between different laboratories and research institutions can significantly accelerate the pace of research and help identify effective treatments and interventions quickly. Therefore, it is crucial to establish effective communication channels and data-sharing platforms to facilitate collaboration between different stakeholders.

### Existing approaches

When looking for solutions to sample collection and processing, you can check [this](https://rdmkit.elixir-europe.org/collecting) documentation.

## Assessment of data quality and data preprocessing

Pre-analytical errors, such as wrong labelling of the specimen, the incorrect specimen collection device, incorrect specimen type are the most frequently reported sources of error (between 48% and 62%) within the clinical laboratory, as stated in this 2021 [publication](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7917444/#s0120), Quality Assurance in the Clinical Virology Laboratory.

> Conducting quality control is an essential step to avoid these kinds of errors. Quality metrics such as read depth, coverage, and base quality scores can help identify errors, artifacts, and biases in the data. Tools such as FastQC, Qualimap, and Picard can be used to assess data quality and generate summary reports. On the other hand, tools such as the FastQScreen and Graf Pop and Sex, can be used to ensure the correct labelling of the samples.

Author: done!

- {% tool "gatk" %}: GATK is a widely used tool for variant calling and genotyping from NGS data. It uses various filtering and quality control options to identify high-quality variants and reduce false positives in the analysis.
- {% tool "picard" %}: Picard is a suite of tools that provides quality control and processing of NGS data, including duplicate read removal, format conversion, and alignment. It is widely used in sequencing data analysis pipelines to improve the quality and reliability of the data.
- {% tool "trimmomatic" %}: Trimmomatic is a tool used for the removal of adapter sequences, low-quality reads, and sequences with ambiguous bases from NGS data. It uses various filtering and trimming options to improve the quality of the sequencing data and prepare it for downstream analysis.
- {% tool "fastqc-screen" %}:  FastQScreen is a quality control tool used to detect contamination in sequencing data (FASTQ files). It screens the input data against a database, formed by the genomes of all of the organisms that could have contaminated the sample, along with PhiX, Vectors or other contaminants commonly seen in sequencing experiments.

> - {% tool "graf-pop" %}: GRAF pop is a software tool that infers the subject ancestry. For the estimation it uses about 100,000 SNPs from genotype datasets, which can be in PLINK and VCF format. 

Reviewer: ```suggestion
- {% tool "graf-pop" %}: GRAF-pop is a software tool that infers the subject ancestry. For the estimation it uses about 100,000 SNPs from genotype datasets, which can be in PLINK and VCF format. 
```

> - {% tool "graf-sex" %}: Determines subject sexes using the genotypes. It’s used to validate the self-reported sexes in phenotype datasets.

Reviewer: ```suggestion
- {% tool "graf-sex" %}: GRAF-sex determines subject sexes using the genotypes. It’s used to validate the self-reported sexes in phenotype datasets.
```

- **Internal human control**: an important aspect is human resource management, for instance, ensuring personnel training, competency, and continuous professional development is necessary.
- **External quality assessment**: external audits are an essential part for ensuring the quality of the procedures. They provide an objective overview that will state any discrepancies between the results and reveal  points of improvement.
- **Procedural documents**:  How an speciﬁc laboratory activity is conducted or how a piece of equipment is has been used must be recorded in standard operating procedures (SOPs) and equipment operating procedures (EOPs). These records will ensure results reproducibility. It’s also important to notice that these records must undergo regular reviews, ensuring that they remain compliant.

> - **Negative and positive controls**: including negative controls it’s important to detect contamination in the reagents, consumables, and general laboratory environment. Including negative controls in each run of diagnostic molecular testing based on RT-PCR, for example, will avoid false positive results resulting from cross-contamination. As well as positive controls for the target sequence/s used in the individual assays, to ensure no false negative results due to reagents failure. For example, the study of a housekeeping human gene would ensure quality of the sample collection procedure: it controls that the sample has human material and quality of the analytical procedure: we check there is no PCR inhibition and the reagents are working properly.

Reviewer: ```suggestion
- **Negative and positive controls**: including negative controls it’s important to detect contamination in the reagents, consumables, and general laboratory environment. Including negative controls in each run of diagnostic molecular testing based on RT-PCR, for example, will avoid false positive results resulting from cross-contamination. As well as positive controls for the target sequence/s used in the individual assays, to ensure no false negative results due to reagents failure. For example, the study of a housekeeping human gene would ensure quality of the sample collection procedure, as it controls that the sample has human material and the quality of the analytical procedure, making it possible to check that there is no PCR inhibition and the reagents are working properly.
```

To ensure the correct  and standard collection of metadata the definition of a metadata schema, beforehand, in an excel or spreadsheet, is a core decision. An interoperable, but yet, customizable metadata scheme can be found in this [publication](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9950378/). 

The mandatory and optional variables must be selected depending on the aim of the study and should include any relevant clinical data such as vaccination state, age, gender, or hospitalisation. More considerations about metadata can be found in the [Data resources page](https://www.infectious-diseases-toolkit.org/data-sources/human-biomolecular-data). 

While completing the forms, the use of ontologies is highly recommended: 
- Infectious Disease Ontology (IDO), a suite of interoperable ontology modules aiming to provide coverage of all aspects of the infectious disease domain. Further details about IDO can be found in this [paper](https://pubmed.ncbi.nlm.nih.gov/34275487/).
- CovidO, an ontology for covid-19 metadata, is described in this [paper](https://link.springer.com/article/10.1007/s11227-023-05509-4).

> Besides sample metadata, assay metadata describing how the results were generated is also necessary. This metadata will depend on the type of assay (antigenic, molecular, target genes of the assay, serological…). And it should include all the used variables on which the results depend on.

Reviewer: ```suggestion
Besides sample metadata, assay metadata describing how the results were generated is also necessary. This metadata will depend on the type of assay (antigenic, molecular, target genes of the assay, serological, etc) and it should include all the used variables on which the results depend on.
```
